extension/js/templates/sidebar.css

```diff
@@ -17,7 +17,7 @@
 }
 
 #package-name {
-  width: 390px;
+  width: 380px;
```

Comment:
do you need to have this with fixed size?

Reply:
Having a 100% width seems to break the searchbar's dimensions. I'll check if there is any better alternative, compared to hardcoding the width.